exporter.lua

```diff
@@ -10,27 +10,50 @@ end
 mist.scheduleFunction(write_state, {}, timer.getTime() + 524, 580)
 
 -- update list of active CTLD AA sites in the global game state
-function enumerateCTLD()
+function countHawkSites()
     local CTLDstate = {}
-    log("Enumerating CTLD")
+    log("Counting the number of HAWK sites.")
+    local hawkCount = 0
     for _groupname, _groupdetails in pairs(ctld.completeAASystems) do
         local CTLDsite = {}
         for k,v in pairs(_groupdetails) do
             CTLDsite[v['unit']] = v['point']
         end
         CTLDstate[_groupname] = CTLDsite
+        hawkCount = hawkCount + 1
     end
     game_state["Theaters"]["Russian Theater"]["Hawks"] = CTLDstate
-    log("Done Enumerating CTLD")
+    log("Done counting Hawks. Found " .. tostring(hawkCount) .. " sites")
+end
+
+
+function sortByCreationTime(a, b)
+  --TODO: handle nil..
```

Comment:
I don't think this will actually be an issue. I found in testing that any load of a mission would drop units from the state previously. And ones that are added to the state upon load or via humans will go through the callback below.